Documentation/Configuration/Typo3ConfVars/SYS.rst

```diff
@@ -950,7 +950,7 @@ linkHandler
     *   `url` (see :t3src:`core/Classes/LinkHandling/UrlLinkHandler.php`)
     *   `email` (see :t3src:`core/Classes/LinkHandling/EmailLinkHandler.php`)
     *   `record` (see :t3src:`core/Classes/LinkHandling/RecordLinkHandler.php`)
-    *   `phone` (see :t3src:`core/Classes/LinkHandling/TelephoneLinkHandler.php`)
```

Comment:
I think these are not words but fixed strings to be used for configuration. Do you have a reference for this change?

Reply:
Ack